ui/app/ducks/gas/gas.duck.js

```diff
@@ -111,26 +115,36 @@ async function basicGasPriceQuery() {
 
 export function fetchBasicGasEstimates() {
   return async (dispatch, getState) => {
+    const isMainnet = getIsMainnet(getState());
+    const currentNetworkId = getCurrentNetworkId(getState());
     const { basicPriceEstimatesLastRetrieved } = getState().gas;
+
+    const previousNetworkID = await getStorageItem('NETWORK_ID');
```

Comment:
I'm not sure this is needed :thinking: 

If we want to store gas estimates per-network, we should use the chain ID in key. e.g. below in `fetchEthGasPriceEstimates` we could call ```getStorageItem(`${chainId}:GAS_ESTIMATE`)```

Otherwise we'd have dependent values that are updated independently and asynchronously, meaning they'd be out-of-sync at least momentarily, which could introduce bugs.